### PR TITLE
1 Random Windows Address and a member test

### PR DESCRIPTION
**Pull request by TheSillyDoggo**

### CODE REVIEW

File: bindings/2.206/GeometryDash.bro [hunk 1]
```diff
@@ -3970,7 +3970,7 @@ class EnhancedGameObject : GameObject {
 	bool m_isNoMultiActivate; // used in platformer stuff
 	// property 99
 	bool m_isMultiActivate;
-	PAD = android32 0x4, win 0x4;
+	PAD = android32 0x4, win 0x4, android64 0x48;
```
Comment (Cvolton) [inline bindings/2.206/GeometryDash.bro:3973]: scary

Comment (matcool) [reply bindings/2.206/GeometryDash.bro:3973]: this is certified scary

Comment (altalk23) [reply bindings/2.206/GeometryDash.bro:3973]: yeah no this sounds like a hack

**Review by altalk23 — CHANGES_REQUESTED** [review]

**Comment by qimiko** [comment]
besides a single windows binding, it does seem like this pull request has become obsolete (proper enhancedgameobject members and tests have been added) so i think it should be closed soon. unless you want to reduce it down to that single windows binding lol

**Pull request closed without merging**